Week2/homework/variable-x.js

```diff
@@ -0,0 +1,21 @@
+//3.1 First, declare your variable x (do not initialize it yet).
+
+let x;
+
+//3.2 Add a console.log statement that explains in words what you think the value of x is, like in this example:
+console.log("the value of my variable x will be: 25");
```

Comment:
I know that the value will be 25, but what is the value right now?

Reply:
The value of x here should be here undefined, I will change it 👍